src/main/resources/i18n/messages.properties

```diff
@@ -119,10 +119,10 @@ FoldTab.64=<html><strong>Default:</strong> Enabled if filename prettifying is en
 FoldTab.65=Monitor played status of files
 FoldTab.66=<html><strong>Default:</strong> Disabled<br><strong>Notes:</strong> Folders must have "Monitor played status of files" enabled to have their contents included in the New Media folder.</html>
 FoldTab.67=Do nothing
-FoldTab.68=Add an overlay to the thumbnail
-FoldTab.69=Hide the file
-FoldTab.70=Move the file to a different folder
-FoldTab.71=Move the file to the recycle/trash bin
+FoldTab.68=Mark media
+FoldTab.69=Hide video
+FoldTab.70=Move file to a different folder
+FoldTab.71=Move file to recycle/trash bin
 FoldTab.72=Fully played action:
```

Comment:
Now we are using 3 different words - media, video and file. Wouldn't it be better to be consistent?